courses/machine_learning/deepdive2/structured/solutions/4a_sample_babyweight.ipynb

```diff
@@ -0,0 +1,2818 @@
+{
```

Comment:
<p>This hasn't changed. If you need a reminder of what we discussed in the team meeting, I'm happy to meet.</p><p> </p><p>The SELECT * FROM and breaking up the query and order by is over-complicating things here. Bonus points for setting count=10 to simplify the future function calls.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/671/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Done</p>